src/PatientChecking/PatientChecking/Controllers/PatientController.cs

```diff
@@ -132,5 +139,113 @@ public IActionResult Detail(int patientId)
 
             return View(model);
         }
+
+        [HttpPost]
+        public IActionResult Update(PatientDetailViewModel model)
+        {
+            var patientDetails = new PatientDetails
+            {
+                PatientId = model.PatientId,
+                FirstName = model.FirstName,
+                MiddleName = model.MiddleName,
+                LastName = model.LastName,
+                FullName = string.IsNullOrEmpty(model.MiddleName) ? model.FirstName + " " + model.LastName : model.FirstName + " " + model.MiddleName + " " + model.LastName,
+                Nationality = model.Nationality,
+                DoB = DateTime.Parse(model.DoB),
+                MaritalStatus = model.MaritalStatus == (int)PatientMaritalStatus.Married,
+                Gender = (PatientGender)model.Gender,
+                EthnicRace = model.Nationality == "Vietnamese" ? model.EthnicRace : null,
```

Comment:
Please make it to Nationality Enum

Reply:
I have created PatientNationality Enum and applied this to code. @vietvo361 Please review this